src/om_bootstrap/nav.cljs

```diff
@@ -146,13 +148,20 @@
                       (d/span {:class "icon-bar" :key 2})
                       (d/span {:class "icon-bar" :key 3})]))))
 
+(sm/defn render-header-and-toggle-btn? :- s/Bool
```

Comment:
Now both the navbar-header and the toggle-button look for these keys. Before, if you supplied :collapsible? true to your Nav props, it wouldn't render the default toggle button unless you added your own, or specified a :toggle-nav-key. Now it also checks for :brand, which I think everyone would supply if they want a collapsible nav.